examples/hacker-news/app/StoryList.re

```diff
@@ -113,12 +113,16 @@ let component = {
         Paging.hook(fetchStories(~makeQuery), resource, 30, hooks);
       (
         hooks,
-        <scrollView
-          onReachedEnd=loadNextPage style=[flex(1.), backgroundColor]>
-          {switch (stories) {
-           | Loading => <activityIndicator style=[flex(1.)] />
-           | stories =>
-             stories
+        switch (stories) {
+        | Loading =>
```

Comment:
This only works once for the initial load. We might want to add another kind of indicator for 'pull-to-refresh', file an issue for that?

Reply:
yes, sure!